.contrib/Parser/DATAS/02 - Outdoor Zones/01 Kalimdor/Felwood/Quests.lua

```diff
@@ -948,6 +948,7 @@ _.Zones =
 					["provider"] = { "n", 15395 },	-- Nafien
 					["coord"] = { 64, 10.4, 77 },
 					["isBreadcrumb"] = true,
+					["u"] = REMOVED_FROM_GAME,
```

Comment:
This is still available in game, but you only can pick it up between 40 and 60 levels [according](https://discord.com/channels/242423099184775169/326533837582630912/643649345215922176) to @slumberine.
And people reported getting it as recently as in [May](https://www.wowhead.com/quest=28768/winterspring#comments:id=3206281).
So instead of marking it as removed might be a good idea to put a note about levels.
```suggestion
					["description"] = "Only available between level 40 and 60.",
```